src/Storage/Storage/Model/Contract/StorageBlobManagement.cs

```diff
@@ -518,7 +515,49 @@ public void SetStorageServiceProperties(StorageServiceType type, ServiceProperti
                         throw new ArgumentException(Resources.InvalidStorageServiceType, "type");
                 }
             }
-            catch (AggregateException e) when (e.InnerException is StorageException)
+            catch (AggregateException e) when (e.InnerException is XSCL.StorageException)
+            {
+                throw e.InnerException;
+            }
+        }
+
+
+
+        /// <summary>
+        /// Get the Table service properties
+        /// </summary>
+        /// <param name="account">Cloud storage account</param>
+        /// <param name="options">Request options</param>
+        /// <param name="operationContext">Operation context</param>
+        /// <returns>The service properties of the specified service type</returns>
+        public XTable.ServiceProperties GetStorageTableServiceProperties(XTable.TableRequestOptions options, XTable.OperationContext operationContext)
```

Comment:
Why is this part of StorageBlobManagement?

Reply:
This is because the cmdlets for serviceproperties currently only has StorageBlobManagement Channel (Now one cmldet only has one Channel). With curently code we can get/set both B/F/Q service properties and Table Service properties. 
If add them to StorageTableManagement, it will take big code change on the Storage cmdlet framework, and make the regression risk higher.